### PR TITLE
New Hindu God and Godess Cheat Sheet

### PR DESCRIPTION
**Pull request by vikashvverma**
Not sure if such cheat sheets are accepted though we can have a discussion.

Screenshot: 

screen shot 2016-02-19 at 12 11 25 am

IA Page: [ https://duck.co/ia/view/hindu_god_cheat_sheet](https://duck.co/ia/view/hindu_god_cheat_sheet)

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Hindu God Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/hindu_god_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
> Not sure if such cheat sheets are accepted though we can have a discussion.

I don't see anything wrong here :+1:

File: share/goodie/cheat_sheets/json/hindu-gods.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    "section_order": [
+        "God",
+        "God (continued)",
+        "Goddess"
+    ],
+    "sections": {
+        "God": [
+            {
+                "key": "Brahma, the Creator",
+                "val": "Brahma is the first member of the Hindu Trinity and is \" the Creator\" because he periodically creates everything in the universe"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: We shouldn't repeat the name -- it looks a little odd.

`First member of the Hindu Trinity and is \"the Creator\" ...`

File: share/goodie/cheat_sheets/json/hindu-gods.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                "key": "Vishnu, the Preserver",
+                "val": "Vishnu is the second member of the Hindu Trinity. He maintains the order and harmony of the universe"
+            },
+            {
+                "key": "Shiva, the Destroyer",
+                "val": "Shiva is the third member of the Hindu Trinity, tasked with destroying the universe in order to prepare for its renewal at the end of each cycle of time"
+            },
+            {
+                "key": "Ganapati, the Remover of Obstacles",
+                "val": "Ganapati, also known as Ganesha, is Shiva’s first son"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: `Shiva's first son` -- we can probably put `(Ganesha)` in the tile

File: share/goodie/cheat_sheets/json/hindu-gods.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    "template_type": "reference",
+    "section_order": [
+        "God",
+        "God (continued)",
+        "Goddess"
+    ],
+    "sections": {
+        "God": [
+            {
+                "key": "Brahma, the Creator",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: I think we should put the 2nd name in `()` or add it to the description

File: share/goodie/cheat_sheets/json/hindu-gods.json (hunk 1 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: sure

Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: Will work on it.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Hindu God Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/hindu_god_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/hindu-gods.json (hunk 2 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: It's done.

File: share/goodie/cheat_sheets/json/hindu-gods.json (hunk 3 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: Done!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Hindu God Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/hindu_god_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Hindu God Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/hindu_god_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/hindu-gods.json (hunk 3 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: It's not really 2nd name.

It's like how we call Obama, Potus.

It's explained in description.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Hindu God Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/hindu_god_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Hindu God Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/hindu_god_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/hindu-gods.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    "id":"hindu_gods_cheat_sheet",
+    "name":"Hindu Gods",
+    "description": "A brief description about Hindu Gods and Goddesses",
+    "metadata": {
+        "sourceName": "Dummies",
+        "sourceUrl": "http://www.dummies.com/how-to/content/hindu-gods-and-goddesses.html"
+    },
+    "aliases": [
+        "hinduism",
+        "hindu gods",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: Remove this one - it's covered by the file name.

File: share/goodie/cheat_sheets/json/hindu-gods.json [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+        "hindu god",
+        "hindu goddesses",
+        "hindu goddess",
+        "hindu god and goddess"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: We could add `hindu gods and goddesses`.

File: share/goodie/cheat_sheets/json/hindu-gods.json [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+                "key": "Indra, the King of Heaven and lord of the gods",
+                "val": "Wields a thunderbolt and is a protector and provider of rain."
+            },
+            {
+                "key": "Surya (Soorya), the sun",
+                "val": "A golden warrior arriving on a chariot pulled by seven white horses"
+            },
+            {
+                "key": "Agni, the fire god",
+                "val": "Holds a special place in Hindu fire ritual to this day as the sacrificer"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: Should this be `fire rituals`?

File: share/goodie/cheat_sheets/json/hindu-gods.json [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+                "key": "Shiva, the Destroyer",
+                "val": "Third member of the Hindu Trinity, tasked with destroying the universe in order to prepare for its renewal at the end of each cycle of time"
+            },
+            {
+                "key": "Ganapati (Ganesha), the Remover of Obstacles",
+                "val": "Shiva’s first son"
+            },
+            {
+                "key": "Rama",
+                "val": "One of the most beloved Hindu gods and is the hero of the Hindu epic called the Ramayana"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: Could we rephrase this to `One of the most beloved Hindu gods and hero of the Ramayana`?

File: share/goodie/cheat_sheets/json/hindu-gods.json (hunk 6 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: `fire ritual` is fine as there are not many.

File: share/goodie/cheat_sheets/json/hindu-gods.json (hunk 7 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: Done!

File: share/goodie/cheat_sheets/json/hindu-gods.json (hunk 5 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: Done

File: share/goodie/cheat_sheets/json/hindu-gods.json (hunk 4 above)
Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/hindu-gods.json]: Removed!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Hindu God Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/hindu_god_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@vikashvverma Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@vikashvverma this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!
